test/test.js

```diff
@@ -0,0 +1,233 @@
+'use strict';
+
+var gulp = require('gulp');
```

Comment:
I'm big fan of the order like: core modules > npm modules > local modules (so path and gulp should switch order) but no need to change that detail right now, maybe just some reminder in future :)